backend/src/main/java/ch/sectioninformatique/template/auth/RegisterDto.java

```diff
@@ -7,6 +7,5 @@
  * @param firstName The user's first name
  * @param lastName The user's last name
  * @param login The user's login identifier
- * @param password The user's password as a character array
  */
-public record SignUpDto (String firstName, String lastName, String login, char[] password) { }
```

Comment:
The file name is "RegisterDto.java" and the record name is "SignUpDtao". We should use the same names for both.